procedures/foreman/branch.md.erb

```diff
@@ -12,98 +14,114 @@
+
+## Release Owner
+
 - [ ] Ensure headline features planned for the release have been merged or push them off to the next release.
-* Translations:
-  - [ ] Update locales in foreman **develop**: `make -C locale tx-update`
+- Translations:
+  - [ ] Update locales in foreman __develop__: `make -C locale tx-update`
   - [ ] Ask plugin authors to start extracting i18n strings and pushing the changes into develop/master git branches so Transiflex can pick it up
   - [ ] Announce string freeze date on discourse and send announcement via https://www.transifex.com/foreman/foreman/announcements/
   - [ ] Update [foreman-dev](https://community.theforeman.org/c/development) with [translations status](https://www.transifex.com/projects/p/foreman/) to encourage 100% translations before release
 
 # During the week before branching
 
+## Release Owner
+
 - [ ] Announce start of stabilization week to discourse [development category](https://community.theforeman.org/c/development).
 - [ ] Request new Hammer CLI release from maintainers if needed.
-* Prepare the manual for the new version:
+- Prepare the manual for the new version:
   - [ ] Change `$next` parameter on [web class](https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/web/manifests/init.pp) to point to the new version number.
   - [ ] Copy [website manual content](https://github.com/theforeman/theforeman.org/tree/gh-pages/manuals) from nightly to <%= release %> and update version numbers mentioned in it.
   - [ ] Clean up deprecation and upgrade warnings from nightly manual.
 - [ ] Refresh unattended templates
   - [ ] Sync the community-templates to Foreman core, by running `script/sync_templates.sh`
   - [ ] Create a <%= @release %>-stable branch in [community-templates](https://github.com/theforeman/community-templates)
 - [ ] Add new languages that are at a [reasonable completion on Transifex](https://www.transifex.com/foreman/foreman/foreman/) to __develop__
-- [ ] Create new GPG key for release. See [GPG_Keys](https://projects.theforeman.org/projects/foreman/wiki/GPG_Keys) if needed.
-  - [ ] [Generate](https://github.com/theforeman/theforeman-rel-eng/blob/master/generate_gpg)
-  - [ ] [Backup](https://github.com/theforeman/theforeman-rel-eng/blob/master/export_gpg_private)
-  - [ ] [Sign](https://github.com/theforeman/theforeman-rel-eng/blob/master/sign_gpg)
-  - [ ] [Upload](https://github.com/theforeman/theforeman-rel-eng/blob/master/upload_gpg)
+
+## Release Engineer
+
+- [ ] [Generate](https://github.com/theforeman/theforeman-rel-eng/blob/master/generate_gpg), [backup](https://github.com/theforeman/theforeman-rel-eng/blob/master/export_gpg_private), [sign](https://github.com/theforeman/theforeman-rel-eng/blob/master/sign_gpg) and [upload](https://github.com/theforeman/theforeman-rel-eng/blob/master/upload_gpg) a GPG key. See [GPG_Keys](https://projects.theforeman.org/projects/foreman/wiki/GPG_Keys) if needed.
```

Comment:
I've had this in the past but then people who followed it missed steps so it was split to explicit steps.

Reply:
so should this be split back up?

Reply:
Please